src/main.c

```diff
@@ -602,13 +553,9 @@ init_tty(void)
 		tty_putp(enter_ca_mode);
 
 	/* Emit enough lines to fit all choices. */
-	SLIST_FOREACH(choice, choices, choices) {
-		choices_count++;
+	for (i = 0; i < (ssize_t)choices.nmemb && i < lines; ++i)
```

Comment:
Given https://github.com/thoughtbot/pick/pull/82/files#r48715231 is correct, should it be `lines - 1` here too?